plugins/base.js

```diff
@@ -79,6 +80,6 @@ class BasePlugin {
 }
 
 BasePlugin.id = 'base';
-BasePlugin.pluginVersion = 2;
+BasePlugin.pluginVersion = 3;
```

Comment:
Bump plugin version due to constructor arity change.

Reply:
I think we should avoid this if possible (as it will likely mean another major).
Can't we pass the creds in the `cfg` object of the plugin constructor rather than adding a separate url arg?

Reply:
Also this file needs a copyright update